lib/aws/http_client.ex

```diff
@@ -0,0 +1,27 @@
+defmodule Aws.HTTPClient do
```

Comment:
I think here is `AWS`

 ```suggestion
defmodule AWS.HTTPClient do
```

Reply:
Oops! Good catch :D